json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -49,3 +49,42 @@ protected function result() {
 
 	}
 }
+
+class Jetpack_JSON_API_Sync_Modify_Settings_Endpoint extends Jetpack_JSON_API_Endpoint {
+	// POST /sites/%s/sync/settings
+	protected $needed_capabilities = array();
```

Comment:
I was having a failure because when we invoke APIs from WPCOM that aren't invoked on behalf of a particular user, then permissions checks fail. I need to figure out a way around that.